src/web/components/AdSlot.tsx

```diff
@@ -5,6 +5,9 @@ import { border, neutral, text } from '@guardian/src-foundations/palette';
 import { textSans } from '@guardian/src-foundations/typography';
 import { from } from '@guardian/src-foundations/mq';
 
+const adSlotStyles = css`
+    position: relative;
```

Comment:
Adding `overflow: hidden;` seems unnecessary as only native ads will be able to interact with the commercial bundle, and we have control of the templates to prevent spilling.